stable/traefik/values.yaml

```diff
@@ -521,3 +521,6 @@ extraVolumes: []
 extraVolumeMounts: []
 #   - name: example-name
 #     mountPath: /path/in/container
+
+networkPolicy:
```

Comment:
Can this be commented out by default?
It will require an extra `if` in the template, but allows for backwards compatability